### ISSUE TITLE
weights initilization method

### ISSUE THREAD

**Issue by DanlanChen** [raghakot/keras-resnet#18]
According to the paper:https://arxiv.org/pdf/1512.03385v1.pdf, it is using  https://arxiv.org/pdf/1502.01852.pdf  to initialize weights, what do you think of it? It also mentions prelu, should we use it, I have such question as my loss is not converging at all...

**Comment by raghakot** [comment]
Try a lower learning rate. I suppose i can provide options to customize activation functions.

**Comment by ColaColin** [comment]
I've also noticed that the cifar10 example, unmodified, does not seem to learn at all. Like it stays at a "perfect" random guess chance of 0.1 acc.

It seems the initialization he_normal is to blame somehow. When I remove it and let it use the default init I see acc of more than 0.2 after not even a single epoch. 

Actually I just made another test and you're right, @raghakot reducing the learning rate is key.
I switched to Adam with a learning rate of 10 times less than the default and that just reached 0.3+ acc in less than one epoch. At the default learning rate Adam fails just like rmsprop: Accuracy stays at exactly 0.1

I'd expect the example to work without tinkering on it however, so that's odd. I am on theano, maybe there is some issue somewhere that makes it converge for some, but not for others?

Is there any reason to use rmsprop over adam in resnets? That might be a stupid question, I have no idea...

**Comment by raghakot** [comment]
Make sure you are using latest version of keras with tf/theano. Should give you ~86% accuracy with no changes. 

You can use Adam. The latest commit switches it to Adam.

**Comment by raghakot** [comment]
This issue is happening with cifar10 example on keras too. Something must have changed with latest updates..

**Comment by raghakot** [comment]
This should now run out of the box after commit: d29048d11705e2f4c372668cea6747f75bdc8959
Reopen if you still see this.

**Comment by kini5gowda** [comment]
Hey,
Isn't the resnet model supposed to give a higher accuracy on cifar 10? Just a doubt to see if there are any changes that can be made to increase accuracy.
